bundles/model/org.eclipse.smarthome.model.sitemap/src/org/eclipse/smarthome/model/validation/SitemapValidator.xtend

```diff
@@ -83,4 +85,34 @@ class SitemapValidator extends AbstractSitemapValidator {
 		}
 	}
 
+	@Check
+	def void checkSetpoints(Setpoint sp) {
+	    if(sp.item === null)
+	    {
```

Comment:
urgh, what happened to your formatter? Don't the brackets feel lonely like this?

Reply:
Seems like the formatter is not a save action for ".xtend" files. I had to run "CTRL + SHIFT + F" to make it work.